tests/test_ml_tools.py

```diff
@@ -156,6 +159,47 @@ def prepare_awkward(self, jets):
     client.close()
 
 
+def test_tensorflow():
+    _ = pytest.importorskip("tensorflow")
+
+    from coffea.ml_tools.tf_wrapper import tf_wrapper
+
+    client = Client()  # Spawn local cluster
+
+    class tf_wrapper_test(tf_wrapper):
+        def prepare_awkward(self, jets):
+            # List of PF candidate features used for computation
+            features = [f"feat{i}" for i in range(1, 19)]
+            # Padding PFCandidate list length to target length (64)
+            cands = ak.pad_none(jets.pfcands, 64, axis=1)
+            # Folding features into inner most axis
+            cands = ak.concatenate(
+                [cands[f][..., np.newaxis] for f in features], axis=2
```

Comment:
Right now, this method is complaining about `None`s existing in the array when running with `dask` arrays. Full trace stack:


```
>       dak_res = tfw(dak_jets)

tests/test_ml_tools.py:195: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
src/coffea/ml_tools/helper.py:380: in __call__
    return self._call_dask(*args, **kwargs)
src/coffea/ml_tools/helper.py:355: in _call_dask
    wrap_meta = wrap(self, *packed_metas)
src/coffea/ml_tools/helper.py:329: in __call__
    (ak_args, ak_kwargs), backends = conv(*ak_args, **ak_kwargs)
src/coffea/ml_tools/helper.py:120: in __call__
    out_args = self.convert(args, backends)
src/coffea/ml_tools/helper.py:115: in convert
    out = self._convert(arg, maybe_backends)
src/coffea/ml_tools/helper.py:104: in _convert
    return arg.__class__(self.convert(val, maybe_backends) for val in arg)
src/coffea/ml_tools/helper.py:104: in <genexpr>
    return arg.__class__(self.convert(val, maybe_backends) for val in arg)
src/coffea/ml_tools/helper.py:115: in convert
    out = self._convert(arg, maybe_backends)
src/coffea/ml_tools/helper.py:110: in _convert
    return call(arg)
/uscms/home/yimuchen/.local/lib/python3.11/site-packages/awkward/typetracer.py:133: in length_one_if_typetracer
    return _length_0_1_if_typetracer(
/uscms/home/yimuchen/.local/lib/python3.11/site-packages/awkward/typetracer.py:81: in _length_0_1_if_typetracer
    layout = function(layout.form, highlevel=False)
/uscms/home/yimuchen/.local/lib/python3.11/site-packages/awkward/forms/form.py:608: in length_one_array
    return ak.operations.ak_from_buffers._impl(
/uscms/home/yimuchen/.local/lib/python3.11/site-packages/awkward/operations/ak_from_buffers.py:150: in _impl
    out = _reconstitute(form, length, container, getkey, backend, byteorder, simplify)
/uscms/home/yimuchen/.local/lib/python3.11/site-packages/awkward/operations/ak_from_buffers.py:382: in _reconstitute
    content = _reconstitute(
/uscms/home/yimuchen/.local/lib/python3.11/site-packages/awkward/operations/ak_from_buffers.py:393: in _reconstitute
    content = _reconstitute(
/uscms/home/yimuchen/.local/lib/python3.11/site-packages/awkward/operations/ak_from_buffers.py:289: in _reconstitute
    content = _reconstitute(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

form = NumpyForm('float64'), length = 0, container = {'node-0': b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00', 'node-1': b'\xff\xff\xff\xff\xff\xff\xff\xff'}
getkey = <function regularize_buffer_key.<locals>.getkey at 0x7fddac5d9120>, backend = <awkward._backends.numpy.NumpyBackend object at 0x7fde6f750b10>, byteorder = '<', simplify = False

    def _reconstitute(form, length, container, getkey, backend, byteorder, simplify):
        if isinstance(form, ak.forms.EmptyForm):
            if length != 0:
                raise ValueError(f"EmptyForm node, but the expected length is {length}")
            return ak.contents.EmptyArray()
    
        elif isinstance(form, ak.forms.NumpyForm):
            dtype = ak.types.numpytype.primitive_to_dtype(form.primitive)
>           raw_array = container[getkey(form, "data")]
E           KeyError: 'None'

```

Reply:
This is now fixed with proper data mangling routine. Basically we need to fix in inner dimension of the array, and not a hard limitation of the dask/awkward. 

Will be adding some additional documentations for common mangling routies.